conductor/blueprints/search/models.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+
+_engine = None
+
+ENABLED_SEARCHES = {
+    'user': {
+        'index': 'users',
+        'doc_type': 'user_profile',
+        '_source': ['idhash', 'name', 'avatar_url', 'datasets']
+    },
+    'package': {
+        'index': 'packages',
+        'doc_type': 'package',
+        '_source': ['id', 'model', 'package', 'origin_url']
+    }
+}
+
+def _get_engine():
+    global _engine
+    if _engine is None:
+        es_host = os.environ['OS_ELASTICSEARCH_ADDRESS']
+        _engine = Elasticsearch(hosts=[es_host], use_ssl='https' in es_host)
+    return _engine
+
+
+def query(kind, size=100, **kw):
+    kind_params = ENABLED_SEARCHES.get(kind)
+    if kind_params is None:
+        return None
+    try:
+        if type(size) is list:
+            size = size[0]
```

Comment:
When `size` can be a list? I usually don't comment much, but I think it would be useful here.